ui/components/ui/box/box.stories.js

```diff
@@ -288,254 +288,265 @@ export const Margin = (args) => {
-        borderColor={COLORS.BORDER_DEFAULT}
+        backgroundColor={BackgroundColor.backgroundDefault}
+        borderColor={BorderColor.borderDefault}
         borderWidth={2}
         marginBottom={1}
       >
-        <Typography color={COLORS.TEXT_DEFAULT}>
-          COLORS.BORDER_DEFAULT
+        <Typography color={TextColor.textDefault}>
+          BorderColor.borderDefault
         </Typography>
       </Box>
       <Box
         padding={3}
-        backgroundColor={COLORS.BACKGROUND_DEFAULT}
-        borderColor={COLORS.BORDER_MUTED}
+        backgroundColor={BackgroundColor.backgroundDefault}
+        borderColor={BorderColor.borderMuted}
         borderWidth={2}
         marginBottom={1}
       >
-        <Typography color={COLORS.TEXT_DEFAULT}>COLORS.BORDER_MUTED</Typography>
+        <Typography color={TextColor.textDefault}>
+          BorderColor.borderMuted
+        </Typography>
       </Box>
       <Box
         padding={3}
-        borderColor={COLORS.PRIMARY_DEFAULT}
+        borderColor={BorderColor.primaryDefault}
         borderWidth={2}
         marginBottom={1}
-        backgroundColor={COLORS.PRIMARY_MUTED}
+        backgroundColor={BackgroundColor.primaryMuted}
       >
-        <Typography color={COLORS.TEXT_DEFAULT}>
-          COLORS.PRIMARY_DEFAULT
+        <Typography color={TextColor.textDefault}>
+          BorderColor.primaryDefault
         </Typography>
       </Box>
       <Box
         padding={3}
-        backgroundColor={COLORS.ERROR_MUTED}
-        borderColor={COLORS.ERROR_DEFAULT}
+        backgroundColor={BackgroundColor.errorMuted}
+        borderColor={BorderColor.errorDefault}
         borderWidth={2}
         marginBottom={1}
       >
-        <Typography color={COLORS.TEXT_DEFAULT}>
-          COLORS.ERROR_DEFAULT
+        <Typography color={TextColor.textDefault}>
+          BorderColor.errorDefault
         </Typography>
       </Box>
       <Box
         padding={3}
-        backgroundColor={COLORS.SUCCESS_MUTED}
-        borderColor={COLORS.SUCCESS_DEFAULT}
+        backgroundColor={BackgroundColor.successDefault}
+        borderColor={BorderColor.successDefault}
         borderWidth={2}
         marginBottom={1}
       >
-        <Typography color={COLORS.TEXT_DEFAULT}>
-          COLORS.SUCCESS_DEFAULT
+        <Typography color={TextColor.textDefault}>
+          BorderColor.successDefault
         </Typography>
       </Box>
       <Box
         padding={3}
-        backgroundColor={COLORS.WARNING_MUTED}
-        borderColor={COLORS.WARNING_DEFAULT}
+        backgroundColor={BackgroundColor.warningMuted}
+        borderColor={BorderColor.warningDefault}
         borderWidth={2}
       >
-        <Typography color={COLORS.TEXT_DEFAULT}>
-          COLORS.WARNING_DEFAULT
+        <Typography color={TextColor.textDefault}>
+          BorderColor.warningDefault
         </Typography>
       </Box>
     </>
   );
 };
+BorderColorStory.storyName = 'BorderColor';
```

Comment:
Can you revert this change on this PR and make it separately?

Reply:
Ah this is actually needed, because the storyname `BorderColor` has the same name as variable`BorderColor.borderDefault`, so i changed the storyname :)